packages/lib-content/src/screens/Home/Community/CommunityContainer.js

```diff
@@ -0,0 +1,26 @@
+import dayjs from 'dayjs'
+import relativeTime from 'dayjs/plugin/relativeTime'
+dayjs.extend(relativeTime)
```

Comment:
PFE's blog feed section uses Moment.js, but that package is [now a legacy package](https://www.npmjs.com/package/moment).

Reply:
The monorepo also uses Luxon, so you might want to consolidate on a single package for handling dates.

Reply:
Was just going to comment the same thing 😆 I think we might be able to simply remove luxon from app-project.

Reply:
I vaguely remember Roger replacing MomentJS with Luzon, but can't remember which component used it. I assume it got removed or refactored at some point.